pyproject.toml

```diff
@@ -31,6 +31,7 @@ pytest-asyncio = "^0.24.0"
 black = ">=24.8.0,<25.0.0"
 flake8 = ">=7.1.1,<8.0.0"
 openai = ">=1.35.12,<2.0.0"
+anthropic = ">=0.39.0,<1.0.0"
```

Comment:
we don't need anthropic in these devdependencies as they are already in the example's dependencies. (in fact, I think the `openai` devdependency can also go, but I'll investigate that at a later time myself)